### ISSUE TITLE
Blank configuration menu with classic Clicked and ElvUI

### ISSUE THREAD

**Issue by themagicteeth** [Snakybo/Clicked#39]
## Describe the bug:
When using with ElvUI (latest version) the configuration menu is completely empty except for a "New" button. Unchecking the Ace3 option under the "Skins" menu in ElvUI fixes the issue.

I have no other addons enabled except ElvUI and Clicked. I have also removed my whole Interface and WTF folders and reinstalled both addons to see if some other addon/setting was causing it. This is completely stock install of ElvUI.

## General information:
[cc_dump_ace3_enabled.txt](https://github.com/Snakybo/Clicked/files/5704956/cc_dump_ace3_enabled.txt)

Clicked Version: 0.12.1-classic
WoW Version: Classic
Did it work before?: Yes, not sure which version. Stopped working sometime after 6d8fc9825559ecf3f0162e607afbbc8866ceaa8e.
Does it occur with all other addons disabled: No

## Expected behavior:
When opening the configuraton while Ace3 skins are enabled in ElvUI, I should be able to configure the addon.

## Reproduction:
1. Install ElvUI Classic (latest version) and Clicked Classic (latest version)
2. Open the Clicked configuration interface, it is blank
3. Open ElvUI configuration, go to "Skins" and under "General" uncheck "Ace3"
4. Reload UI, and open the Clicked configuration. Addon can now be configured.

## Screenshots
Before disabled Ace3 in ElvUI:
![ace](https://user-images.githubusercontent.com/18600239/102391106-e128e800-3f9a-11eb-8881-a1ddaeed60fe.jpg)

After disabled Ace3 in ElvUI:
![noace](https://user-images.githubusercontent.com/18600239/102391098-df5f2480-3f9a-11eb-90a7-9ff2a5ea7231.jpg)

**Comment by Snakybo** [comment]
Hm, I suspect there's an error happening. Can you type `/console scriptErrors 1` and then `/reload`? If there's an error a popup box will appear. After you're done you can type `/console scriptErrors 0` to disable the popup box again.

**Comment by themagicteeth** [comment]
Two messages popped up after enabling `/console scriptErrors 1` and opening the Clicked confiuration:

````
Message: ...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:1481: attempt to index field 'backdrop' (a nil value)
Time: Thu Dec 17 12:10:22 2020
Count: 1
Stack: ...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:1481: attempt to index field 'backdrop' (a nil value)
[C]: ?
...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:1481: in function `?'
...erface\AddOns\Clicked\Libs\AceGUI-3.0\AceGUI-3.0.lua:108: in function <...erface\AddOns\Clicked\Libs\AceGUI-3.0\AceGUI-3.0.lua:97>
...erface\AddOns\Clicked\Libs\AceGUI-3.0\AceGUI-3.0.lua:146: in function `Create'
Interface\AddOns\Clicked\Config\Bindings.lua:1476: in function <Interface\AddOns\Clicked\Config\Bindings.lua:1473>
Interface\AddOns\Clicked\Config\Bindings.lua:1538: in function `OpenBindingConfig'
Interface\AddOns\Clicked\Core\Clicked.lua:32: in function `OnClick'
...\AddOns\Clicked\Libs\LibDBIcon-1.0\LibDBIcon-1.0.lua:144: in function <...\AddOns\Clicked\Libs\LibDBIcon-1.0\LibDBIcon-1.0.lua:142>

Locals:
````

````
Message: ...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:377: attempt to index local 'self' (a nil value)
Time: Thu Dec 17 12:10:22 2020
Count: 1
Stack: ...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:377: attempt to index local 'self' (a nil value)
[C]: ?
...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:377: in function <...Clicked\Widgets\AceGUIContainer-ClickedTreeGroup.lua:374>

Locals: 
````

**Comment by Snakybo** [comment]
Thanks, fixed in 0.13.2
